listener/wal_transaction.go

```diff
@@ -217,7 +218,12 @@ func (w *WalTransaction) CreateActionData(
 			valueType: rel.Columns[num].valueType,
 			isKey:     rel.Columns[num].isKey,
 		}
-		column.AssertValue(row.Value)
+		if row.IsUnchangedToastedValue && len(oldRows) > num {
+			oldColumns[num].isUnchangedToastedValue = true
+			column.AssertValue(oldRows[num].Value)
+		} else {
+			column.AssertValue(row.Value)
+		}
```

Comment:
Do we want something similar to this below, in `CreateEventsWithFilter` (line 273)

Reply:
No, setting `column.AssertValue(oldRows[num].Value)` here ensures `val.value` already has the correct value on line 273.

Reply:
🙇 I wasn't sure if this code was feeding into that code somehow. Merci!